bin/get_current_version.sh

```diff
@@ -5,10 +5,11 @@
 export CURRENT_VERSION
 CURRENT_VERSION=v$(git describe --tags --always | tr -d .)
 
-regex='^v[0-9]{3}([ab][0-9]{1,}|rc[0-9]{1,})?$'
+export KUBENOW_REGEX
+KUBENOW_REGEX='^v[0-9]{3}([ab][0-9]{1,}|rc[0-9]{1,})?$'
```

Comment:
As per our discussion, based on shellcheck output, it's recommended to split exporting and initialization in the same line.